Developer-guide/Simplified-Development-Workflow.md

```diff
@@ -152,12 +152,16 @@ To submit your change for review, run the rfc.sh script,
 		$ ./rfc.sh
 
 The script will ask you to enter a bugzilla bug id. Every change
```

Comment:
The script will prompt for a valid bug number in the [Red Hat Bugzilla](https://bugzilla.redhat.com/)